.travis.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+echo "Building for Linux..."
+GOOS=linux   GOARCH=amd64 go build
```

Comment:
Do we need to specify architecture, here?

Reply:
Actually, no. Will drop that.